.github/workflows/preview-docs.yml

```diff
@@ -43,10 +41,35 @@ jobs:
     if: |
       github.event.action == 'closed' &&
       github.event.pull_request.head.repo.full_name == github.repository
-    uses: Qiskit/gh-actions/.github/workflows/code-engine-cleanup.yml@main
-    with:
-      code_engine_project: qiskit-docs-preview
-      docker_image_name: qiskit-docs-preview
-    secrets:
-      ibmcloud_account: ${{ secrets.IBMCLOUD_ACCOUNT }}
-      ibmcloud_api_key: ${{ secrets.IBMCLOUD_API_KEY }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set PREVIEW_TAG
+        run: echo "PREVIEW_TAG=pr-${{ github.event.pull_request.number }}" >> $GITHUB_ENV
+      - name: Set APP_NAME and DOCKER_IMAGE_TAG
+        run: |
+          echo "APP_NAME=qiskit-docs-preview-${{ env.PREVIEW_TAG }}" >> $GITHUB_ENV
+          echo "DOCKER_IMAGE_TAG=icr.io/community-digital/qiskit-docs-preview:${{ env.PREVIEW_TAG }}" >> $GITHUB_ENV
+      - name: Install IBM Cloud CLI
+        run: |
+          curl -o IBM_Cloud_CLI_2.20.0_amd64.tar.gz -L https://download.clis.cloud.ibm.com/ibm-cloud-cli/2.20.0/IBM_Cloud_CLI_2.20.0_amd64.tar.gz
```

Comment:
We used to install the latest version of Linux, which fails. This is evidence for why I think this job hasn't been running: https://github.com/Qiskit/documentation/issues/1034.

Or actually maybe it's been running, but just failing without us knowing. TBD>